vita3k/config/src/config.cpp

```diff
@@ -290,7 +290,7 @@ ExitCode init_config(Config &cfg, int argc, char **argv, const Root &root_paths)
         config_desc.add_options()
             ("archive-log,A", po::bool_switch(&command_line.archive_log), "Makes a duplicate of the log file with TITLE_ID and Game ID as title")
             ("backend-renderer,B", po::value(&command_line.backend_renderer), "Renderer backend to use, either \"OpenGL\" or \"Vulkan\"")
-            ("color-surface-debug, C", po::bool_switch(&command_line.color_surface_debug),"Save color surfaces")
+            ("color-surface-debug,C", po::bool_switch(&command_line.color_surface_debug), "Save color surfaces")
```

Comment:
oups :D

Reply:
What oops?

Reply:
my fault lol, i have added this line, forget format :D

Reply:
NP XD

Should be fixed now.